config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/ServiceLoader.java

```diff
@@ -6,29 +6,31 @@
 
 package com.ca.apim.gateway.cagatewayconfig.bundle.loader;
 
-import com.ca.apim.gateway.cagatewayconfig.beans.Service;
 import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
 import com.ca.apim.gateway.cagatewayconfig.beans.Folder;
+import com.ca.apim.gateway.cagatewayconfig.beans.Service;
+import com.ca.apim.gateway.cagatewayconfig.beans.WSDL;
 import com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes;
 import com.ca.apim.gateway.cagatewayconfig.util.string.EncodeDecodeUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.w3c.dom.Element;
 import org.w3c.dom.Node;
 import org.w3c.dom.NodeList;
 
 import javax.inject.Singleton;
-
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
 import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.getSingleChildElement;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.nodeList;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.*;
 
 @Singleton
 public class ServiceLoader implements BundleEntityLoader {
 
+    static final String KEY_VALUE_SOAP = "soap";
+    static final String KEY_VALUE_SOAP_VERSION = "soapVersion";
+    static final String TAG_VALUE_POLICY = "policy";
+    static final String TAG_VALUE_WSDL = "wsdl";
+
     @Override
     public void load(Bundle bundle, Element element) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 24 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=130&issues=AWcUvrYBkEeD8QRkNhVh&open=AWcUvrYBkEeD8QRkNhVh)

[ISSUE_KEY]: AWcUvrYBkEeD8QRkNhVh

Reply:
Try to split this into 2 or 3 methods so we workaround this warning.